src/dom/html/WindowRe.re

```diff
@@ -95,6 +95,28 @@ module Impl (T: { type t; }) => {
 
 
   external setOnLoad : t_window => (unit => unit) => unit = "onload" [@@bs.set]; /* use addEventListener instead? */
+
+  module Image = {
```

Comment:
This module really describes an `HTMLImageElement` (see https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image) and should therefore be in a module named accordingly.

Perhaps it should be aliased as `Image` somewhere to better conform to expectations, but I'm not really sure where it would make sense to place it.

Reply:
Did you forget to remove these from `Window`?

Reply:
I sure did! Should be ready now 👍